src/pages/dashboard.tsx

```diff
@@ -54,36 +54,43 @@ export default function DashboardPage(props: PageProps) {
     signIn,
   } = React.useContext(UserDataContext);
 
+  let showIgnored = userSettings.showIgnored;
+
   const lastViewedModuleURL = moduleIDToURLMap[lastViewedModuleID];
   const activeModules: ActiveItem[] = React.useMemo(() => {
     return Object.keys(userProgressOnModules)
       .filter(
         x =>
           (userProgressOnModules[x] === 'Reading' ||
             userProgressOnModules[x] === 'Practicing' ||
-            userProgressOnModules[x] === 'Skipped') &&
+            userProgressOnModules[x] === 'Skipped' ||
+            (showIgnored && userProgressOnModules[x] === 'Ignored')) &&
           moduleIDToSectionMap.hasOwnProperty(x)
       )
       .map(x => ({
         label: `${SECTION_LABELS[moduleIDToSectionMap[x]]}: ${
           moduleIDToName[x]
         }`,
         url: moduleIDToURLMap[x],
-        status:
-          userProgressOnModules[x] === 'Skipped' ? 'Skipped' : 'In Progress',
+        status: userProgressOnModules[x] as
+          | 'Skipped'
+          | 'Reading'
+          | 'Practicing'
+          | 'Ignored',
       }));
   }, [userProgressOnModules]);
```

Comment:
need to also add `showIgnored` to the dependency array so that the memoized function reruns when showIgnored is changed

Reply:
(I fixed this and merged)